Makefile

```diff
@@ -72,7 +72,7 @@ clearstates:
 	
 kaboom: clearstates spincontainer install
 
-devchecknotest: install golangcilint generate tfproviderlint tflint terrafmtlint importfmtlint
+devchecknotest: verifycontent install golangcilint generate tfproviderlint tflint terrafmtlint importfmtlint
```

Comment:
You can remove this from devcheck below so it doesn't run twice

Reply:
removed from devcheck